spec/requests/profiles_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe 'Profiles' do
```

Comment:
Thank you for adding these. I still want to understand more what the difference between request specs and controller specs are because this is exactly how i'd do a controller spec.

Reply:
My understanding is that the RSpec team recommends folks phase out controller specs in favour of request specs, the primary reason being that behind the scenes a controller spec is more of a unit test while a request spec is more of an integration test and a "more realistic" testing scenario (according to the RSpec blog, anyway...).

I take a lot of queues from Jason Swett on this topic: https://www.codewithjason.com/use-controller-request-specs-rails-dont/

I've been writing a lot of request specs lately for reason 3 on my day job app, so just in the habit.

Reply:
Love it! Thanks for sharing